src/scheduler/workers/main.new.ts

```diff
@@ -0,0 +1,26 @@
+import { main as unpublishWeeklyTask } from "./unpublish/weekly";
+import { main as unpublishDayBeforeTask } from "./unpublish/dayBefore";
+import { main as unpublishDayTask } from "./unpublish/day";
```

Comment:
I feel like it would be better to make `main` the default export and then the imports can be written as 

```suggestion
import unpublishWeeklyTask from "./unpublish/weekly";
import unpublishDayBeforeTask from "./unpublish/dayBefore";
import unpublishDayTask  from "./unpublish/day";
```

Reply:
Or, you could rename `main` in the relevant files to the name you are giving it here? If that makes sense.